### PR TITLE
feat: Add @mohanamisra as a contributor

### LINKED ISSUE #553: Feature: Add @mohanamisra as a contributor
### Description

As part of the Intro to Open Source course, we are encouraged to contribute to this repo

### Suggested solution

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Getting Started Instructions

- [X] I agree to follow this project's Getting Started instructions

### PR DESCRIPTION
**Pull request by mohanamisra**
## Description
Adding user @mohanamisra (yours truly) as a contributor to this repository, as required in the Intro to Open Source course by OpenSauced.

## What type of PR is this? (check all applicable)

- [x] 🤝 Add a contributor
- [ ] 📝 Documentation Update

## Related Issues
Closes #553 

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section

- [x] ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?

- [x] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [x] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot (Required for PR Review)

  
image

## [optional] What GIF best describes this PR or how it makes you feel?
  
68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f776174747061642d6d656469612d736572766963652f53746f7279496d6167652f386568486d71734a3135344d58773d3d2d3839373430373335352e313631353131313165363838653639643535343336363133343637372e676966

### CODE REVIEW

**Comment by BekahHW** [comment]
Hey @mohanamisra there's a slight change you need in the issue `Closes Issue #553`, but you need to remove the word "issue" because it breaks the key word `closes` from the issue number. So you'll need to edit that part of your PR description to `closes #553`.

**Comment by mohanamisra** [comment]
> Hey @mohanamisra there's a slight change you need in the issue `Closes Issue #553`, but you need to remove the word "issue" because it breaks the key word `closes` from the issue number. So you'll need to edit that part of your PR description to `closes #553`.

Made the required change. Awaiting review!

File: .idea/.gitignore
Comment (adiati98) [inline .idea/.gitignore:1]: Please remove this file.

File: .idea/guestbook.iml
Comment (adiati98) [inline .idea/guestbook.iml:1]: Please remove this file.

File: .idea/modules.xml
Comment (adiati98) [inline .idea/modules.xml:1]: Please remove this file.

File: .idea/vcs.xml
Comment (adiati98) [inline .idea/vcs.xml:1]: Please remove this file.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hi @mohanamisra,
Thank you for your contribution! ✨ 

However, there are new files that we don't want to have here. Please remove them.

Let us know when it's fixed and ready for a review.
Thank you. 🙂

**Comment by mohanamisra** [comment]
> However, there are new files that we don't want to have here. Please remove them.
> 
> Let us know when it's fixed and ready for a review. Thank you. 🙂

I apologise for that !
 Removed the extra files as requested.

**Review by adiati98 — APPROVED** [review]
Well done and thank you for your contribution, @mohanamisra! ✨ 

If you haven't, you can join our [Community](https://github.com/orgs/open-sauced/discussions/1).
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕🍴

well done GIF

**Review by BekahHW — APPROVED** [review]

**Pull request merged**